packages/backend-lib/src/journeys.ts

```diff
@@ -335,7 +335,11 @@ group by event, node_id;`;
             childEdge: 100,
           },
           sendRate: 0,
-          channelStats: {} as MessageChannelStats,
+          channelStats: {
+            ...(node.variant.type === ChannelType.Sms && {
```

Comment:
This is a bit weird. You're doing a spread operator on a value of `boolean | { type: ChannelType.Sms }`. Doing a spread operator on a boolean doesn't throw here I think, but it's still a bit hard to understand.

How about we just make `channelStats` an optional property,

```typescript
const MessageNodeStats = Type.Object({
  type: Type.Literal(NodeStatsType.MessageNodeStats),
  proportions: Type.Object({
    childEdge: Type.Number(),
  }),
  sendRate: Type.Number(),
  channelStats: Type.Optional(MessageChannelStats),
});
```

You'll need to adapt a bit of the upstream code, but should be cleaner I think. Then you can just leave the `channelStats` portion here undefined.